codedocs/doxygen.conf

```diff
@@ -213,7 +213,7 @@ SUBGROUPING            = YES
 # Private class members and static file members will be hidden unless 
 # the EXTRACT_PRIVATE and EXTRACT_STATIC tags are set to YES
 
-EXTRACT_ALL            = NO
+EXTRACT_ALL            = YES
```

Comment:
Is this acceptable to everyone? It was the only way I could find to generate documentation for the `pdns::checked_*` functions.